app/containers/MassEnergizeSuperAdmin/Community/Profile/About.js

```diff
@@ -64,8 +64,14 @@ class About extends React.Component {
     let oldLoadingCSVs = this.state.loadingCSVs;
     this.setState({ loadingCSVs: oldLoadingCSVs.concat(endpoint) });
 
-    const csvResponse = await apiCallFile('/downloads.' + endpoint,
-      { community_id: community.id });
+    const body = {};
+    const dot = endpoint.indexOf('.');
+    if (dot > 0) {
```

Comment:
what computation is happening here?  what is `dot` supposed to tell us?  maybe a comment would really help.

Reply:
Clarified with a comment, slight update to code